src/app/modules/connect-to-wallet/connect-buttons/connect-buttons.component.html

```diff
@@ -6,7 +6,7 @@
   </button>
 </div>
 <div class="my-3">
-  <button mat-raised-button (click)="connectToWalletConnect()"
+  <button mat-raised-button *ngIf="false" (click)="connectToWalletConnect()"
```

Comment:
I would comment it out, or add a variable in the UI or .env - enable wallet login. It is more explicit and future developers will know.

Reply:
I would chose adding a variable to enable this button.